### PR TITLE
Change username options added to developer options

### PR DESCRIPTION
**Pull request by BarAgam**
Change username options added to developer options

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -122,9 +122,14 @@
   <string name="dialog_go_to_long_lat_label">Destination</string>
   <string name="unknown_location">Unknown location</string>
   <string name="menu_use_utm_title">Use UTM</string>
+
+  <!--Developer options-->
   <string name="menu_clear_vl_cache_title" translatable="false">Clear VL Cache</string>
+  <string name="menu_change_username_title" translatable="false">Change username</string>
```
Comment (teamagam5) [inline app/src/main/res/values/strings.xml:128]: If you use the existing class as I mentioned, you can spare this.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -122,9 +122,14 @@
...
   <string name="clear_cache_successful_message" translatable="false">All layers successfully deleted</string>
   <string name="clear_cache_failure_message" translatable="false">Couldn\'t delete all layers</string>
+  <string name="alert_dialog_change_username_title" translatable="false">Change username</string>
```
Comment (teamagam5) [inline app/src/main/res/values/strings.xml]: Could be a ref to `menu_change_username_title`

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -122,9 +122,14 @@
...
+  <string name="alert_dialog_change_username_positive_button_text" translatable="false">OK</string>
```
Comment (teamagam5) [inline app/src/main/res/values/strings.xml]: Could use `android.R.string.ok`

File: domain/src/main/java/com/teamagam/gimelgimel/domain/utils/PreferencesUtils.java [hunk 4]
```diff
@@ -32,6 +32,10 @@ public void toggleCoordinateSystemPrefs() {
     mUserPreferencesRepository.setPreference(USE_UTM_PREF_KEY, !shouldUseUtm());
   }
 
+  public void setUsername(String username) {
```
Comment (teamagam5) [inline domain/src/main/java/com/teamagam/gimelgimel/domain/utils/PreferencesUtils.java]: Nice.
The `SetUsernameAlertDialogBuilder` should use this too.
(The current impl. use prefs directly I think)

**Review by teamagam5 — APPROVED** [review]

File: domain/src/main/java/com/teamagam/gimelgimel/domain/utils/PreferencesUtils.java (hunk 4 above)
Comment (BarAgam) [reply domain/src/main/java/com/teamagam/gimelgimel/domain/utils/PreferencesUtils.java]: I actually prefer to delete it and keep the direct implementation.
It's in the domain and used only by the app.

**Pull request merged**